iteration/iteration_test.go

```diff
@@ -433,11 +426,10 @@ func (s *TestIterationRepository) TestExistsIteration() {
 
 		repo := iteration.NewIterationRepository(s.DB)
 		level0IterationName := "Top level iteration"
-		userActive := false
 		i1 := iteration.Iteration{
 			Name:       level0IterationName,
 			SpaceID:    space.ID,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c